### ISSUE TITLE
Show character limit upfront

### ISSUE THREAD

**Issue by abitrolly** [huggingface/chat-ui#333]
There are many frustrations with https://huggingface.co/chat giving incomplete answers. One is listed on this tracker - #287. Others are in `chat-ui` community https://huggingface.co/spaces/huggingchat/chat-ui/discussions/195 and in model specific chats https://huggingface.co/OpenAssistant/oasst-sft-6-llama-30b-xor/discussions/44

It would be nice to show actual model limitations upfront. I am not sure if there is a standard config setting for that, but here is the one for the current model. https://huggingface.co/OpenAssistant/oasst-sft-6-llama-30b-xor/discussions/53

![image](https://github.com/huggingface/chat-ui/assets/8781107/d4303e40-0257-4f18-8efb-65ea3ece0719)

**Comment by nsarrazin** [comment]
Yeah thanks for bringing it up to the surface, it's indeed a common issue. Not 100% sure but I think the length limit is for tokens rather than characters, which make it hard to display client-side. I did have plans for a "continue generation" button: https://github.com/huggingface/chat-ui/issues/280 so I think that would help with that as well.

**Comment by nsarrazin** [comment]
Yeah the goal is to have a button that just picks up where it last left. Haven't added it yet though!
